hack/push_updates.sh

```diff
@@ -0,0 +1,140 @@
+#!/bin/bash
+#
+# Push updates that need to be vendored in other repositories.
+
+if ! which hub 2>/dev/null 1>&2; then
+    echo "This tool requires the hub command line app." 1>&2
+    echo "Refer to https://github.com/github/hub for installation instructions." 1>&2
+    exit 1
+fi
+
+function show_help() {
+    cat <<EOF
+push_updates.sh [-h]
+push_updates.sh [-b BRANCH] [-n] [-r REPO] [-s SHA] [-t TITLE]
+
+  -b BRANCH
+    Set the release branch to be updated. Defaults to "master".
+
+  -h
+    Shows this help message.
+
+  -n
+    Dry run mode. Do not push or create a pull-request.
+
+  -r REPO
+    Update the given repository. May be repeated.
+    Defaults to updating all cluster-api-provider repositories:
+EOF
+
+    for r in $DEFAULT_REPOS; do
+        echo "      $r"
+    done
+
+cat <<EOF
+
+  -s SHA
+    The SHA from machine-api-operator repository to push.
+    Defaults to HEAD of master.
+
+  -t TITLE
+    The commit and pull-request title to use. Defaults to
+    a message that includes the SHA.
+
+Without any arguments, creates a pull request for each repository to
+update the vendored version of machine-api-operator to refer to the
+current HEAD commit on the master branch.
+
+EOF
+}
+
+DEFAULT_REPOS="
+cluster-api-provider-baremetal
+cluster-api-provider-gcp
+"
```

Comment:
These are missing multiple repos. 
The full list should contain:
- cluster-api-provider-gcp
- cluster-api-provider-aws
- cluster-api-provider-azure
- cluster-api-provider-openstack
- cluster-api-provider-baremetal
- cluster-api-provider-ovirt

Reply:
Added